prow/create-deps-cm.sh

```diff
@@ -105,10 +107,18 @@ get_options() {
       cm_key="$2"
       shift 2
       ;;
+    -l | --local)
+      local=1
+      shift
+      ;;
     -i | --interactive)
       interactive=1
       shift
       ;;
+    --dry-run)
```

Comment:
`--dry-run=true` is a better/safer default than the current `--dry-run=false`.

If so then `--confirm` might be a better name choice since it naturally defaults to the safe mode.